src/style.css

```diff
@@ -1,3 +1,63 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  --color-blue: #00acdc;
+  --gap: 20px;
```

Comment:
Usually it is better to define CSS variables inside `:root` or `html` selectors, it is not a mistake though, just a recommendation. The same for `font-family`

Reply:
Thanks, it's better practice

Reply:
Interesting thing,   "margin: 0;" don't works into :root

Reply:
Cause :root doesn't have any wrappers for margins